packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -785,11 +786,27 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
     // we don't have event for it, but want to keep the event handling interface, event is empty.
     _.invoke(this.props, 'onSelectedChange', {}, { ...this.props, value })
   }
+
+  /**
+   * if there is no value we use the placeholder value
+   * otherwise, for single selection we convert the value with itemToString
+   * and for multiple selection we return empty string, the values are rendered by renderSelectedItems
+   */
+  private getSelectedItemAsString = (value: ShorthandValue): string => {
+    const { itemToString, multiple, placeholder } = this.props
+    return (_.isArray(value)
+    ? value.length < 1
+    : !value)
+      ? placeholder
+      : multiple
+      ? ''
+      : itemToString(value)
+  }
 }
 
 Dropdown.slotClassNames = {
   container: `${Dropdown.className}__container`,
+  triggerButton: `${Dropdown.className}__trigger-button`,
+  itemsList: `${Dropdown.className}__items-list`,
   selectedItems: `${Dropdown.className}__selected-items`,
 }
-
-export default Dropdown
```

Comment:
Oops, you removed the export of the Dropdown.

Reply:
fixed